### PR TITLE
Fix svace issues

### PR DESCRIPTION
**Pull request by jeongarmy**

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@sangam-swami @abhishek-samsung She make changes on compression. Could you review?

**Comment by sunghan-chang** [comment]
@taehwan-hwang Could you review changes on ble manager for SVACE?

File: os/tools/mksyscall.c [hunk 1]
```diff
@@ -543,10 +543,12 @@ int main(int argc, char **argv, char **envp)
 		case '?':
 			fprintf(stderr, "Unrecognized option: %c\n", optopt);
 			show_usage(argv[0]);
+			break;
 
 		case ':':
 			fprintf(stderr, "Missing option argument, option: %c\n", optopt);
 			show_usage(argv[0]);
+			break;
```
Comment (sunghan-chang) [inline os/tools/mksyscall.c:551]: how about default?

File: os/tools/mksyscall.c [hunk 2]
```diff
@@ -543,10 +543,12 @@ int main(int argc, char **argv, char **envp)
...
 
 		default:
```
Comment (abhishek-samsung) [inline os/tools/mksyscall.c:553]: I think we will not reach default at all as the above cases cover all possibilities, if so can we remove the default case?

Comment (jeongarmy) [reply os/tools/mksyscall.c:553]: I think we need to keep this because of unexpected inputs.

**Comment by sunghan-chang** [comment]
@sangam-swami @abhishek-samsung Could you confirm the changes on compress is ok?

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**